package.json

```diff
@@ -20,6 +20,7 @@
     "@types/chai": "^4.2.0",
     "@types/mocha": ">=9.1.0",
     "@types/node": ">=12.0.0",
+    "bignumber.js": "^9.1.1",
```

Comment:
You can use BigInt or ether's BigNumber. No need to bring an additional library

Reply:
This repository uses ethers v6 and `BigNumber` is no longer available with v6.
https://docs.ethers.org/v6/migrating/#migrate-bigint

And I don't see any functions related to math operations in BigInt (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt)

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#operators

Reply:
BigInt is exactly the reason why BigNumber is not a part of ethers anymore 😉